app/public/html/choose-account.twig

```diff
@@ -3,43 +3,77 @@
+                        </option>
+                    {% endfor %}
+                </select>
+            </div>
+            <div class="form-group">
+                <label for="firefly_account">Firefly Account</label>
+                <select class="form-control" name="firefly_account" id="firefly_account" required>
+                    {% for account in firefly_accounts %}
+                        <option value="{{ account.id }}">
+                            {% if account.iban %}{{ account.iban }} &mdash; {% endif %}
+                            {{ account.name }}
+                        </option>
+                    {% endfor %}
+                </select>
+            </div>
+
+            <div class="form-group">
+                <label for="date_from">Import start date</label>
+                <input class="form-control" name="date_from" id="date_from" type="date" required
+                    value="{{ default_from_date | date('Y-m-d') }}">
+            </div>
+            <div class="form-group">
+                <label for="date_to">Import end date</label>
+                <input class="form-control" name="date_to" id="date_to" type="date" required
+                    value="{{ default_to_date | date('Y-m-d') }}">
+            </div>
+
+            <button type="submit" class="btn btn-primary">Collect statements from your bank</button>
+        </form>
+    {% else %}
+        <h1>Importing from Bank Account IBAN {{bank_account_iban}} into Firefly Account ID {{firefly_account_id}}</h1> 
+
+        <p>Continuing to next step in 2 seconds.</p>
+
+        <form name="next-form" action="." method="post">
+            <input type="hidden" name="step" value="{{ next_step }}">
+            <input type="text" name="bank_account" id="bank_account" value="{{ bank_account_index }}" readonly>
+            <input type="text" name="firefly_account" id="firefly_account" value="{{ firefly_account_id }}" readonly>
+
+            <div class="form-group">
+                <label for="date_from">Import start date</label>
+                <input class="form-control" name="date_from" id="date_from" type="date" required
+                    value="{{ default_from_date | date('Y-m-d') }}">
+            </div>
+            <div class="form-group">
+                <label for="date_to">Import end date</label>
+                <input class="form-control" name="date_to" id="date_to" type="date" required
+                    value="{{ default_to_date | date('Y-m-d') }}">
+            </div>
+
+            <button type="submit" class="btn btn-primary">Collect statements from your bank</button>
+        </form>
+
+        <script>
+            window.onload = function() {
+                setTimeout(function() {
+                    document.forms["next-form"].submit();
+                }, 2000);
+            }
+        </script>
```

Comment:
Hm, this is a bit awkward:  
This leaves the user with 2 seconds to change the dates.  

How about adding the from and to date (e.g. in the form 'now - 1 month') to the config too? Then, we auto-skip this form iff all four values (bank_account, firefly_account, date_from, date_to) are provided.